public/src/__tests__/updateEnrollment.test.tsx

```diff
@@ -1,18 +1,53 @@
-import { Enrollment, Enrollment_UserStatus, User } from "../../proto/qf/types_pb"
+import { Enrollment, Enrollment_UserStatus, Enrollments, User } from "../../proto/qf/types_pb"
 import { createOvermindMock } from "overmind"
 import { config } from "../overmind"
 import { createMemoryHistory } from "history"
 import React from "react"
 import Members from "../components/Members"
 import { Route, Router } from "react-router"
 import { Provider } from "overmind-react"
-import { initializeOvermind } from "./TestHelpers"
 import { render, screen } from "@testing-library/react"
-import { Timestamp } from "@bufbuild/protobuf";
+import { PartialMessage, Timestamp } from "@bufbuild/protobuf";
+import { MockData } from "./mock_data/mockData"
+import { EnrollmentRequest, Void } from "../../proto/qf/requests_pb"
+import { CallOptions } from "@bufbuild/connect"
+import { Response } from "../client"
+import { initializeOvermind } from "./TestHelpers"
 
 
 describe("UpdateEnrollment", () => {
-    const mockedOvermind = initializeOvermind({})
+    const mockedOvermind = initializeOvermind({}, {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        getEnrollments: jest.fn(async (request: PartialMessage<EnrollmentRequest>, _?: CallOptions | undefined): Promise<Response<Enrollments>> => {
```

Comment:
It looks a bit awkward, but if this is how it needs to be then that's how it needs to be... just wish it was more readable.

Reply:
I've introduced a `mock` function that should make mocking individual rpc methods easier and more readable.